scss/components/_title-bar.scss

```diff
@@ -42,7 +42,7 @@ $titlebar-icon-spacing: 0.25rem !default;
 
     @if $global-flexbox {
       display: flex;
-      justify-content: space-between;
+      justify-content: flex-start;
```

Comment:
So shall we insert this in a variable ? 
@rafibomb?  
and keep flex start? 
And some docs about it?